util/dvsim/dvsim.py

```diff
@@ -625,6 +631,14 @@ def parse_args():
         print(version)
         sys.exit()
 
+    # Check conflicts
+    # interactive and remote, r
+    if args.interactive and args.remote:
```

Comment:
Hmm, `args.remote` likely needs to be deprecated, since we do not use it anymore.

Reply:
Other places use the `args.remote`. We may create another PR to remove all args.remote usage then. In my opinion, we'd better keep this part in this PR.